apps/builder/app/routes/rest.assets.$name.tsx

```diff
@@ -15,11 +20,37 @@ export const action = async (
 
   try {
     if (request.method === "POST" && request.body !== null) {
-      const asset = await uploadFile(
-        params.name,
-        request.body,
-        createAssetClient()
-      );
+      let body = request.body;
+
+      const contentType = request.headers.get("Content-Type");
+
+      if (contentType?.includes("application/json")) {
```

Comment:
why includes and not === ?

Reply:
`Content-Type: application/json; charset=UTF-8`